packages/vendor/bundle/webpack.config.js

```diff
@@ -43,7 +38,7 @@ const config = {
   },
 
   resolve: {
-    modules: [nodeModulesPath, vendorCoreNodeModulesPath],
+    modules: ['node_modules'],
```

Comment:
This should fix the build issue and also solve a bug.
With the previous configurations, `node_modules` can be loaded only if they are in the project `vendor/node_modules` or in `vendor-core/node_modules` but won't be able to load a module if it's in `vendor-core/node_modules/some-module/node_modules`.

Now, if `some-module` would ask for `some-other-module`, webpack will first check at `vendor-core/node_modules/some-module/node_modules/some-other-module` and then will go back at the file directories to search to the module.